### DISCUSSION TITLE
请问taro支持淘宝小程序吗？

### DISCUSSION THREAD

**Discussion by paomian7fenshu** [NervJS/taro#16082] [Q & A]
没有搜索到相关案例，请问有支持过淘宝小程序的吗？

**Comment by honlyHuang** [comment]
暂时还没有，你可以通过[拓展编译平台](https://docs.taro.zone/docs/platform-plugin/)的方式，自己做一个插件。https://docs.taro.zone/docs/platform-plugin/
